lib/management/configuration.js

```diff
@@ -115,11 +119,23 @@ function patchConfiguration(instanceId, newConf, log, cb) {
                         };
                     }
                     break;
-                case 'location-do-spaces-v1':
+                case 'location-scality-ring-s3-v1':
+                    pathStyle = true; // fallthrough
```

Comment:
Reporting the style comment about using a mapping object rather than relying on fallthrough of a switch.

Reply:
It's a good cleanup, we'll follow up on it